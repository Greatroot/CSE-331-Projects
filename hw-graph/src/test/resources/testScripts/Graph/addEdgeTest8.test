# Test pointing a node to itself | should return true and add the edge

CreateGraph graph
AddNode graph n1
AddEdge graph n1 n1 e1
