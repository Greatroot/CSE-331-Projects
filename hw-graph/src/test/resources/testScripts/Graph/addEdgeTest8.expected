# Test pointing a node to itself | should return true and add the edge

created graph graph
added node n1
added edge e1 from n1 to n1 in graph
